frontend/src/app/services.ts

```diff
@@ -53,7 +53,7 @@ const handleDownloadPDF = (response) => {
 
         const alink = document.createElement("a");
         alink.href = window.URL.createObjectURL(blob);
-        alink.download = `${filename}.pdf`;
+        alink.download = `${filename}`;
```

Comment:
?

Reply:
It is explained in the commit message :)
```
frontend: Fix downloaded files containing duplicate filetype
Downloaded files were named e.g. `file.pdf.pdf`
```